### DISCUSSION TITLE
How to create custom new panel left side in existing ?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3977] [General]
Hi ,

I need two panel left and right side to my project. Already right side available by default, how to create left side panel customisable ? It will good if it is configurable.

I don't know how to add new panel left

Please anyone help me for this.

Thanks advance.

**Comment by souri84** [comment]
hi, 

by adding new sidebar manually and attach traits etc directly into this

for example

```
<div class="flex">
<div class="w-1/3" id="sidebar"></div> // your new sidebar
<div class="w-2/3" id="main"><div id="gjs"></div> // right sidebar already included</div>
</div>
```

**Reply by san-123** [reply]
@artf ,

How you attach in trait ? Please sample code. Here I tried but not working

Please guide me, How to add new panel in existing?
